Cargo.toml

```diff
@@ -89,6 +89,10 @@ harness = false
 [workspace]
 members = ["components/salsa-macro-rules", "components/salsa-macros"]
 
+[[example]]
+name = "calc"
+crate-type = ["bin"]
```

Comment:
Can you tell me more why this change is necessary? I can run `cargo run --example calc` on `master`.

Reply:
oh cool, sorry nevermind then!